components/Nav/SearchWithAlgolia.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react'
+import Router from 'next/router'
+
+import Search from './Search'
+
+let docsearch
+if (process.browser) {
+  docsearch = require('docsearch.js')
+}
+
+class SearchWithAlgolia extends Component {
+  isDocs = true
+
+  componentWillMount() {
+    this.isDocs = process.browser && Router.pathname.startsWith('/docs')
+  }
+
+  componentDidMount() {
+    if (process.browser && typeof docsearch !== 'undefined') {
+      docsearch({
+        apiKey: '79886fb59ad3ebe2002b481cffbbe7cb',
```

Comment:
Is this safe to be committed?

Reply:
dunno, but I suppose you can see the email they've sent me in the issue. They clearly said use it in the client side.

Reply:
Yeah if it's client-side it has to be safe to commit.